Sources/Opus/Opus.Decoder.swift

```diff
@@ -73,30 +109,58 @@ public extension Opus.Decoder {
 // MARK: Private decode methods
 
 extension Opus.Decoder {
-	private func decode(_ input: UnsafeBufferPointer<UInt8>, to output: UnsafeMutableBufferPointer<Int16>) throws -> Int {
-		let decodedCount = opus_decode(
-			decoder,
-			input.baseAddress!,
-			Int32(input.count),
-			output.baseAddress!,
-			Int32(output.count),
-			0
-		)
+	private func decode(_ input: UnsafeBufferPointer<UInt8>,
+	                    to output: UnsafeMutableBufferPointer<Int16>,
+	                    packetSize: Int32?) throws -> Int
+	{
+		var decodedCount: Int32 = 0
+		if let size = packetSize {
+			decodedCount = opus_custom_decode(
+				decoder,
+				input.isEmpty ? nil : input.baseAddress,
+				size,
```

Comment:
Why is `packetSize` passed as an argument here (and other `decode` methods) when it should just be equal to `Int32(input.count)`?

Reply:
packet size is needed for the custom decoder to know how much data it has. 
In the case of a dropped packet, input is nil, but the encoder still needs to know how much data got dropped

Reply:
See comment at the top-level of this PR regarding dropped packets.

If the decoder was initialized with a custom frame size, then either use that, or have `decodeDroppedPacket` accept an (optional?) frame size argument.

Reply:
Custom frame size and compressed packet size are not the same thing. Custom encode / decode need to know both in order to work.